documentation/SupportingDocuments/API_Documentation/User Stories/Edge Cloud APIs UserStory.md

```diff
@@ -0,0 +1,181 @@
+I want to be able to put my application available in the selected Edge Cloud Zone in a simpler manner providing both artifacts and metadata for instantiating my application succesfully at the same time I want to be able to terminate running instances of my application on a particular Edge Cloud Zone and also to remove information of the whole application itself ".  
+
+This story includes the whole journey from the Edge Cloud APIs with a couple of examples about the use of these APIs to improve the deployment experience of the APs:
+## Edge Cloud APIs
+| API | Description|
+|-------|------|-------|
+|Application Endpoint Discovery(AED) |The Application Discovery API extends beyond the capabilities of the Simple Edge Discovery API by not only locating the nearest Edge Cloud Zone but also directly linking to the application endpoints within those zones. |
+|Edge Application Management(EAM)|Edge Application Management API allows API consumers to manage the Life Cycle of an Application and to Discover Edge Cloud Zones. 
+|Simple Edge Discovery(SED)|The Simple Edge Discovery API returns the name of the closest operator Edge Cloud Zone to a particular user device.
+|Traffic Influence(TI)|The TI API provides the capability to establish the optimal routing, in terms of latency, in a specific geographical area, between the user Device, e.g. the user’s smartphone, and the optimal EAS instance nearby.
+
+To further information about each API, review the User story of each  one, which is available in the corresponding repository. 
+The purpose of this user story is to have a general picture of the life cycle management of the Edge cloud family APIs.
+
+## Actors, Roles, and Scope
+| Actor | Role | Scope |
+|-------|------|-------|
+|Application Provider (AP) |API Consumer | API Requester |
+|Edge Platform (EP)|Edge Cloud Provider | API Publisher |
+|Edge Service Provider (ESP)|Edge Cloud Provider|API Publisher
+|Operator|Communications Service Provider|API Publisher
+|User Client(UC)|API Consumer|API Requester
+
+Note: the role of Edge Service Provider may be played by either the MNO or Cloud Service Providers (Hyperscalers).
+
+
+#### Scenario 1: Request triggered by the AP and the Hyperscaler will host the App.
+
+
+| Item | Description | Support Qualifier |
+|----|----|----|
+|Summary|Edge Service Providers offers an Edge  platform that exposes computing and networking services in a given territory with several levels of abstraction being the Edge Cloud Zone the lowest one. Edge Service Providers offers a service through EdgeCloud APIs that permits application providers to deploy instances of an application in one or several Edge Cloud Zones, and to Applications Users the posibility to chose the most appropriate Edge Cloud Zone available at a given moment| M |
+|Pre-conditions|The AP has a resource usage agreement with the Operator| M |
+|Begins when|The user client request access to the application| M |

```

Comment:
I think the statement "user client request access to the application" needs to be rephrased as it is not clear what exactly it is trying to do in terms of API invocation.
Also in the next line i.e., Step 1, it says AP invokes discovery while on line 41 it is about user client so a connection seems to be broken.

Also i think there should be step for onboarding application too which will be referred in other steps.